rhel/SPECS/victoriametrics.spec

```diff
@@ -0,0 +1,59 @@
+%define debug_package %{nil}
+
+%define copying() \
+%if 0%{?fedora} >= 21 || 0%{?rhel} >= 7 \
+%license %{*} \
+%else \
+%doc %{*} \
+%endif
+
+%global provider        github
+%global provider_tld    com
+%global project         VictoriaMetrics
+%global repo            VictoriaMetrics
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+%global commit          be6ae4b5e755df57db135b77e749067c8822a96a
+%global shortcommit     %(c=%{commit}; echo ${c:0:7})
+
+Name:           percona-victoriametrics
+Version:        1.40.1
```

Comment:
That line will be `sed`'ed too

Reply:
There is no victoriametrics submodule at `pmm-submodules` repo, so `sed` shouldn't touch this file.